### PR TITLE
0837126 - snapshot url path updated 

### PR DESCRIPTION
**Pull request by cs-pranali-chavan**
https://mantis.fortinet.com/bug_view_page.php?bug_id=0837126
snapshot url path updated in info.json

### CODE REVIEW

File: widget/info.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
         "certified": "Yes",
         "publisher": "Fortinet",
         "snapshots": [
-            "https://repo.fortisoar.fortinet.com/content-hub/socManagement-1.0.0/latest/images/soc-management.png"
+            "https://repo.fortisoar.fortinet.com/fsr-widgets/socManagement-1.0.0/latest/images/soc-management.png"
```
Comment (cs-pradeep-gurav) [inline widget/info.json]: https://repo.fortisoar.fortinet.com/fsr-widgets/socManagement-1.0.0/images/soc-management.png

File: widget/info.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
         "certified": "Yes",
         "publisher": "Fortinet",
         "snapshots": [
```
Comment (cs-pratap-nikam) [inline widget/info.json:8]: remove **/latest/**
URL Should be -
https://repo.fortisoar.fortinet.com/fsr-widgets/socManagement-1.0.0/images/soc-management.png

**Review by cs-pratap-nikam — CHANGES_REQUESTED** [review]
remove /latest/
URL Should be -
https://repo.fortisoar.fortinet.com/fsr-widgets/socManagement-1.0.0/images/soc-management.png

**Review by cs-pratap-nikam — APPROVED** [review]
Looks Good

**Pull request merged**